### DISCUSSION TITLE
Is mandatory the maintenance of DDBB in BirdNet-Pi station????

### DISCUSSION THREAD

**Discussion by jimenoma** [mcguirepr89/BirdNET-Pi#1231] [Q&A]
Hi Everybody from Spain:
I have installed a BirdNet-Pi in Madrid Spain (Piscis) with a Raspberry Pi 4 8Gb 64G SSD.
The system is running properly, and recognition a lot of birds.
But i think the BBDD is full, in "Species Stats" the range date start in 2024-06-26 and finish in 2024-07-25, not the date of today.
Is need to delete de BBDD or any kind of maintenance? a guideline with instructions would be a useful help.
I have searched in the forum but i have found nothing.
Thanks in advance.
Marco A.
PD. Sorry by my english.

**Comment by Nachtzuster** [comment]
Could you try restarting the Streamlit Statistics service to see if the data appears?
Tools->Services
